### PR TITLE
2.26.0 Release notes

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

File: Release Notes/Release Notes v2.26.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+This is the general availability release for the 2.26.0 version of the driver.
+
+The main new features in 2.26.0 include:
+
++ Support SelectMany inside Project/Select - [CSHARP-5081](https://jira.mongodb.org/browse/CSHARP-5081)
++ Support Dictionary.ContainsValue in LINQ queries - [CSHARP-2509](https://jira.mongodb.org/browse/CSHARP-2509)
++ Support string concatenation of mixed types - [CSHARP-5071](https://jira.mongodb.org/browse/CSHARP-5071)
++ Support serialization of Memory and ReadOnlyMemory structs - [CSHARP-4807](https://jira.mongodb.org/browse/CSHARP-4807)
++ OIDC: support for GCP Identity Provider - [CSHARP-4610](https://jira.mongodb.org/browse/CSHARP-4610)
++ Nuget packages signing - [CSHARP-5050](https://jira.mongodb.org/browse/CSHARP-5050)
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.26.0.md:12]: Implement signing of NuGet packages

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Minor rewording

File: Release Notes/Release Notes v2.26.0.md (hunk 1 above)
Comment (BorisDog) [reply Release Notes/Release Notes v2.26.0.md]: Thanks, done.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**